lib/telegraph/node_element.rb

```diff
@@ -0,0 +1,59 @@
+module Telegraph
+  class NodeElement
+    AVAILABLE_TAGS = %w{
```

Comment:
`AVAILABLE` to whom? Why not just `TAGS`?
And I'd group them in several short lines.